dali/pipeline/pipeline.cc

```diff
@@ -215,14 +213,6 @@ void Pipeline::Init(int max_batch_size, int num_threads, int device_id, int64_t
         std::min(lowest_cuda_stream_priority, highest_cuda_stream_priority);
     const auto max_priority_value =
         std::max(lowest_cuda_stream_priority, highest_cuda_stream_priority);
-    DALI_ENFORCE(
```

Comment:
Can you remove the lines 206/208 up to here?
```
int lowest_cuda_stream_priority = 0, highest_cuda_stream_priority = 0;
    // do it only for the GPU pipeline
    if (device_id != CPU_ONLY_DEVICE_ID) {
      CUDA_CALL(cudaDeviceGetStreamPriorityRange(&lowest_cuda_stream_priority,
                                                 &highest_cuda_stream_priority));
    }
    const auto min_priority_value =
        std::min(lowest_cuda_stream_priority, highest_cuda_stream_priority);
    const auto max_priority_value =
        std::max(lowest_cuda_stream_priority, highest_cuda_stream_priority);
```
I guess we don't need it anymore.

Reply:
Follow up? `max_num_streams` (found only in native code) is also unused, so I'll remove it as well.